cli/src/test/java/org/projectnessie/tools/catalog/migration/cli/HadoopCLIMigrationTest.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2023 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.tools.catalog.migration.cli;
+
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+
+public class HadoopCLIMigrationTest extends AbstractCLIMigrationTest {
+
+  @BeforeAll
+  protected static void setup() {
+    sourceCatalogProperties = "warehouse=" + warehouse1.toAbsolutePath() + ",type=hadoop";
```

Comment:
OK. I will move `CatalogMigrationUtil` to `api` module from `cli`. 
So that I can use it in the test module of both `api` and `cli` and keep all configurations centralized.